front/pages/api/w/[wId]/assistant/agent_configurations/index.ts

```diff
@@ -352,9 +352,6 @@ export async function createOrUpgradeAgentConfiguration({
     }
   }
 
-  if (maxToolsUsePerRun === undefined) {
```

Comment:
I would keep that error for now but maybe I'm missing something?

Reply:
We have a default downstream. not useful to error here IMHO since we're just keeping the default everywhere for now.